torch/utils/_stats.py

```diff
@@ -3,8 +3,13 @@
 # AND SCRUB AWAY TORCH NOTIONS THERE.
 import collections
 import functools
+from typing import OrderedDict
 
-simple_call_counter = collections.OrderedDict()
+simple_call_counter: OrderedDict[str, int] = collections.OrderedDict()
+
+def count_label(label):
+    prev = simple_call_counter.setdefault(label, 0)
+    simple_call_counter[label] = prev + 1
```

Comment:
You could use a defaultdict and just += 1 here.

Reply:
IDK why @voznesenskym didn't make this a defaultdict, I was minimizing changes here

Reply:
Sure, that's fine.